### PR TITLE
add some images

### PR DESCRIPTION
**Pull request by tadeegan**

### CODE REVIEW

File: src/app/fortune/fortune.component.ts [hunk 1]
```diff
@@ -19,7 +19,7 @@ import { Router, ActivatedRoute } from '@angular/router';
 })
 export class FortuneComponent {
   image: string;
-  allImages = ['1000', 'bus', 'carfixed', 'down', 'dts', 'earliest', 'friend', 'harriet', 'inLA', 'kravmaga', 'nexttime', 'notbackintime', 'notsure', 'ohyah', 'outoftown', 'softmaybe', 'vivek', 'wenthome', 'whoknows', 'wildride', 'wrong'];
+  allImages = ['1000', 'bus', 'carfixed', 'could-be', 'down', 'dts', 'earliest', 'friend', 'harriet', 'inLA', 'kravmaga', 'nexttime', 'notbackintime', 'notsure', 'ohyah', 'outoftown', 'softmaybe', 'vivek', 'wenthome', 'whoknows', 'wildride', 'wrong', 'live'];
```
Comment (atscott) [inline src/app/fortune/fortune.component.ts]: Could you keep these sorted (move live after kravmaga)?

Comment (tadeegan) [inline src/app/fortune/fortune.component.ts]: Why isnt this autogenerated?

Comment (atscott) [inline src/app/fortune/fortune.component.ts]: Good question. Why haven't you filed an issue for it?

Comment (atscott) [inline src/app/fortune/fortune.component.ts]: You may revert this file. The list is now auto-generated.

**Comment by tadeegan** [comment]
PTAL

**Pull request merged**